docs/workflows/build-pages-with-storybook.md

```diff
@@ -131,7 +133,8 @@ Once that configuration is complete, we can set the mock values in a specific st
 <CodeSnippets
   paths={[
     'react/app-story-with-mock.js.mdx',
-    'vue/app-story-with-mock.js.mdx'
+    'vue/app-story-with-mock.js.mdx',
+    'angular/app-story-with-mock.ts.mdx'
```

Comment:
```suggestion
    'angular/app-story-with-mock.ts.mdx',
```

Reply:
Fixed with the latest commit. Thank you for checking it :)